cider/tests/nextgen/benchmark/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2022 Intel Corporation.
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+set(DEP_LIBS ${EXECUTE_TEST_LIBS} benchmark::benchmark nextgen)
```

Comment:
hmm, how about we all use folly for benchmark in this lib? https://github.com/intel/BDTK/blob/e20868c128c76625024d3822536914e8142e1e65/cider-velox/benchmark/CMakeLists.txt#L55

Reply:
seems follybenchmark does not support template. maybe we should always use google::benchmark with its more rich features.